semantic/flatbuffers_test.go

```diff
@@ -196,7 +201,114 @@ func getUnaryOpFlatBuffer() (*semantic.Package, []byte) {
 func getFnExprFlatBuffer() (*semantic.Package, []byte) {
```

Comment:
You could refactor this function with the one below it to get rid of some duplicate code that constructs the assignment statement in the flatbuffer, as well as the Go structure. But I'll leave that up to you.

Reply:
Done!